.github/workflows/build.yml

```
@@ -12,4 +12,4 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: gradle/wrapper-validation-action@v1
-      - run: ./gradlew build javadocJar checkMappings mapNamedJar --stacktrace
+      - run: ./gradlew checkLicenses build javadocJar checkMappings --stacktrace
```

Hmm isn't `checkLicenses` a dependency of `check` (and thus `build`)?

---

```suggestion
      - run: ./gradlew build javadocJar checkMappings --stacktrace
```
It indeed is according to gradle build scan. Optimizing.